angular/src/app/components/users-panel/users-panel.component.html

```diff
@@ -25,6 +25,11 @@ <h6 id="property-title">Description:</h6>
                     (nameChange)="changeProjectDescription($event)"
                 >
                 </app-edit-name-input>
+
+                <div style="display: flex; flex-direction: column; justify-content: center">
+                    <button class="button hollow" (click)="openTaggit()">Open Map in Taggit</button>
```

Comment:
what about: "Open map in Taggit" 


----------------


not for this PR but would good to have small Taggit logo in react version.